docs/collections.md

```diff
@@ -93,92 +186,53 @@ tags: post
 This will not be available in `collections.all` or `collections.post`.
 ```
 
-## Tag Syntax
-
-You can use a single tag, as in the above example OR you can use any number of tags for the content, using YAML syntax for a list.
-
-### A single tag: cat
-
-```
----
-tags: cat
----
-```
-
-This content would show up in the template data inside of `collections.cat`.
-
-### Multiple tags, single line
 
-```
----
-tags: ['cat', 'dog']
----
-```
-
-This content would show up in the template data inside of `collections.cat` and `collections.dog`.
-
-### Multiple tags, multiple lines
-
-```
----
-tags:
-  - cat
-  - dog
----
-```
 
-This content would show up in the template data inside of `collections.cat` and `collections.dog`.
+## Data structure of collection items
 
-### Collection Item Data Structure
-
-{% codetitle "Liquid, Nunjucks", "Syntax" %}
-
-{% raw %}
-```
-<ul>
-{%- for post in collections.post -%}
-  <li>{{ post.data.title }}</li>
-{%- endfor -%}
-</ul>
-```
-{% endraw %}
-
-Note in the above example that we output the `post.data.title` value? Similarly, each collection item will have the following data:
+Eleventy populates each collection item with a set of useful information. Each item in a collection has access to the following variables:
 
 * `inputPath`: the full path to the source input file (including the path to the input directory)
 * `fileSlug`: {% addedin "0.5.3" %} Mapped from the input file name, useful for permalinks. Read more about [`fileSlug`](/docs/data/#fileslug).
 * `outputPath`: the full path to the output file to be written for this content
 * `url`: url used to link to this piece of content.
 * `date`: the resolved date used for sorting. Read more about [Content Dates](/docs/dates/).
-* `data`: all data for this piece of content (includes any data inherited from layouts)
+* `data`: all front matter data for this piece of content (includes any data inherited from layouts).
```

Comment:
hmm, I don’t believe this is accurate—is it? it should be all data including data inherited from layouts and directory data files, global data files, etc etc. Everything from the data cascade

Reply:
Ah yes. I must’ve short-circuited to this conclusion. Reverting this change.